starter_code/views/partials/artistsPartials.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="row">
```

Comment:
Te dejo el código que hemos estado comentando
<ul>
  {{#each body.artists.items}}
  <li>
    <h1>{{this.name}}</h1>
    <img src="{{this.images.0.url}}">
    <a href="/albums/{{this.id}}">Show albums</a>
  </li> {{/each}}
</ul>

Reply:
`<ul>
  {{#each body.artists.items}}
  <li>
    <h1>{{this.name}}</h1>
    <img src="{{this.images.0.url}}">
    <a href="/albums/{{this.id}}">Show albums</a>
  </li> {{/each}}
</ul>`